ONSdigital/design-system
# New feature issue template for GitHub

### What is the context of this PR?
- Updated the 'feature request' github issues template
- Remove empty 'Research on this...' sections from all components and patterns
- Changed 'Design System Forum' sections to 'Help improve this ...' with new content to include instructions for sharing UR findings

### How to review
- Check issues template
- Check bottom of all DS pages for new content
